File: AI/code_01/library/specific/Matlab/FileNamesMore_v01.m
function [MonteCarlo,MonteCarloTest,L,K,M,PdBTxRx,Init,Iter,GTS,Name1,Name2]=FileNamesMore_v01(datar,selected,iter)
TtTR=3;                          % Training data to test data ratio
x=selected(iter);
MonteCarlo=datar(x,1);           % Training data
MonteCarloTest=MonteCarlo/TtTR;  % Test data
%MonteCarloTest=10000;           % Number of test data is fixed to a number
L=datar(x,2);                    % Number of access points
K=datar(x,3);                    % Number of users
M=datar(x,4);                    % Number of antennas at an access point
PdBTxRx=datar(x,5);              % Transmit power of AP in dBm
Init=datar(x,6);                 % Number of random initializations
Iter=datar(x,7);                 % Number of iterations
GTS=datar(x,8);                  % Number of access point search groups
Name1=sprintf('L%dK%dM%d',L,K,M); % File name extension 
Name2=['T' mat2str(PdBTxRx) '_In' mat2str(Init) '_I' mat2str(Iter) '_G' mat2str(GTS)]; % File name extension 

File: AI/code_01/library/specific/Python/FileNamesMore_v01.py
from os import listdir, system
from pathlib import Path
def FileNamesMore(datar,x,mfilenameAlg,mfilenameVer,results,BSelecAlg,BSelecAlgVer,BSelecAlgName):         
    TtTR=3                              # Training data to test data ratio
    MonteCarlo=datar['MC'][x]           # Training data
    MonteCarloTest=int(MonteCarlo/TtTR) # Test data
    #MonteCarloTest=10000              # Number of test data is fixed to a number
    L=datar['L'][x];                    # Number of access points    
    K=datar['K'][x];                    # Number of users
    M=datar['M'][x];                    # Number of antennas at an access point
    TxdBm=datar['Tx dBm'][x];           # Transmit power of AP in dBm
    
    Init=datar['Init'][x];              # Number of random initializations
    Iter=datar['Iter'][x];              # Number of iterations
    GTS=datar['GTS'][x];                # Number of access point search groups
    Name2='T'+str(TxdBm)+'_In'+str(Init)+'_I'+str(Iter)+'_G'+str(GTS)  # File name extension 
    filezip='L'+str(L)+'K'+str(K)+'M'+str(M)+'_'+Name2+'_'+BSelecAlgName+'_'+BSelecAlg+'_'+BSelecAlgVer+'.zip'    # Zip file name 
    FileNamex = [x for x in listdir() if x.endswith(".xlsx")]
    FileName=FileNamex[0]
    while len(FileNamex)>1:  # Close the Output_v0X.xlsx file
            print('Please, close the',FileName,'file and then press ENTER \n')      
            system('pause')
            FileNamex = [x for x in listdir() if x.endswith(".xlsx")]
    SheetName='MC'+str(MonteCarlo)+'_'+'MC'+str(MonteCarloTest)
    ResultsFolder='.\\Results'+'\\'+SheetName+'\\'+mfilenameAlg+'\\'+mfilenameVer+'\\' +str(results)
    Path(ResultsFolder).mkdir(parents=True,exist_ok=True)
    NetworkParameters=datar[['L','K','M','Tx dBm','Init','Iter','GTS']][x:x+1]       
    NetworkParameters.insert(3,'Network','L'+str(L)+'-'+'K'+str(K)+'-'+'M'+str(M))    
    return MonteCarlo,MonteCarloTest,L,K,M,Name2,filezip,FileName,SheetName,ResultsFolder,NetworkParameters